lib/osutil/isdir.go

```diff
@@ -14,13 +14,14 @@ import (
 
 // IsDir returns true if base and every path component of name up to and
 // including filepath.Join(base, name) is a directory (and not a symlink or
-// similar). Base and name must both be clean and name must be relative to
+// similar), but returns true if the path is inaccessible,
```

Comment:
This is broken. :) Maybe just `... is a directory or non-existent (and not a symlink`?